libminifi/include/properties/Properties.h

```diff
@@ -51,13 +57,13 @@ class Properties {
   // Set the config value
   void set(const std::string &key, const std::string &value) {
     std::lock_guard<std::mutex> lock(mutex_);
-    properties_[key] = value;
+    properties_[key] = PropertyValue{value, true};
     dirty_ = true;
   }
   // Check whether the config value existed
-  bool has(std::string key) const {
+  bool has(const std::string& key) const {
     std::lock_guard<std::mutex> lock(mutex_);
-    return properties_.count(key) > 0;
+    return properties_.find(key) != properties_.end();
```

Comment:
This seems like a question of preference. As either way is correct, I think every author I free do decide which one to use, but I wouldn't change it. It just adds extra diffs to the code review without value.

Reply:
reverted